### PR TITLE
Add memo and value to invoice details screen

### PR DESCRIPTION
**Pull request by ncoelho**
Add memo and value to invoice details screen. When is a receive payment, there's no "description" field.

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/etta1n4zr70ycm7qrwagg35heg

File: screen/lnd/lndViewInvoice.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
 } from '../../BlueComponents';
 import PropTypes from 'prop-types';
 import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
-import { Icon } from 'react-native-elements';
+import { Text, Icon } from 'react-native-elements';
```
Comment (marcosrdz) [inline screen/lnd/lndViewInvoice.js]: Import Text from react-native.

File: screen/lnd/lndViewInvoice.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { View, Dimensions, ScrollView, BackHandler, InteractionManager } from 'react-native';
+import { View, Dimensions, ScrollView, BackHandler, InteractionManager, TouchableOpacity, } from 'react-native';
```
Comment (marcosrdz) [inline screen/lnd/lndViewInvoice.js]: Remove the last comma

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/2rxprq4233y3vaj85mqw8at5nw

**Comment by marcosrdz** [comment]
rebased.

**Comment by GladosBlueWallet** [comment]
This was a triumph. I'm making a note here: HUGE SUCCESS.

 [android] https://appetize.io/app/3a0f84j9tdre0qq9xchd2bh100

**Pull request merged**